### PR TITLE
Use token icon color palette for the celebration confetti

### PR DESCRIPTION
**Pull request by thinkstpaul**
## Proposed changes
- Add custom confetti colours for each token type using the icon's palette

## How to test
- Pull the branch
- Create different tokens and confirm the confetti matches the token icon colours

## Demo

https://github.com/thinkst/canarytokens/assets/145110595/9602ba0f-a8ac-4137-ab03-fcf4e3077c26

### CODE REVIEW

File: frontend_vue/src/utils/confettiEffect.ts [hunk 1]
```diff
@@ -25,8 +26,8 @@ export function launchConfetti() {
   myConfetti({
     particleCount: 100,
     spread: 160,
-    origin: { y: 0.9 },
-    colors: ['#F2059F', '#04D9B2', '#80C7F2'],
+    origin: { y: .4 },
+    colors: TOKEN_COLOR_PALETTES[token_type],
```
Comment (vittoriaThinkst) [inline frontend_vue/src/utils/confettiEffect.ts:30]: I guess we should add a fallback, in case no colour palette is provided

Comment (thinkstpaul) [reply frontend_vue/src/utils/confettiEffect.ts:30]: Sure, makes sense

File: frontend_vue/src/components/constants.ts [hunk 2]
```diff
@@ -111,3 +111,31 @@ export const ENTRA_ID_FEEDBACK_MESSAGES = {
...
+  slow_redirect: ['#FCA40F', '#424242', '#F9B219', '#E98906', '#565656'],
+  cmd: ['#1AABEE', '#026696', '#9CE5FD'],
+  azure_id: ['#38C0F0', '#0070C6', '#31AFE9', '#0B5BA5'],
+  ms_excel: ['#33c481', '#45946a', '#108d44', '#185c37', '#107c41'],
+  ms_word: ['#2B7CD3', '#1553BA', '#41A5EE', '#185ABD', '#2B6ECC'],
+  svn: ['#829ECB', '#89A5D1', '#E0E9F5'],
+  smtp: ['#E6E7E8', '#535B67', '#9FF18D', '#4EDA82', '#85EA89'],
+  sql_server: ['#BBC1C8', '#FF0000', '#CFD6DD', '#C22625'],
+  signed_exe: ['#434850', '#8BEEAB', '#3CD57F', '#69E285'],
+  adobe_pdf: ['#FE534A', '#FD705D', '#FF0000', '#FF5042', '#FF5042'],
```
Comment (vittoriaThinkst) [inline frontend_vue/src/components/constants.ts:133]: we might use the same as sql_server
`  adobe_pdf: ['#BBC1C8', '#FF0000', '#CFD6DD', '#C22625'],
`

File: frontend_vue/src/components/constants.ts [hunk 3]
```diff
@@ -111,3 +111,31 @@ export const ENTRA_ID_FEEDBACK_MESSAGES = {
...
+  windows_dir: ['#D2961B', '#FDDD81', '#F0C153', '#E5AA2E', '#F3C75E'],
```
Comment (vittoriaThinkst) [inline frontend_vue/src/components/constants.ts:134]: here we might just make the color gap stronger, but keeping the brownish/yellow

`  windows_dir: ['#D2961B', '#FDDD81', '#E5AA2E', '#7a5c09'],
`

Comment (thinkstpaul) [reply frontend_vue/src/components/constants.ts]: I've updated

File: frontend_vue/src/components/constants.ts (hunk 2 above)
Comment (thinkstpaul) [reply frontend_vue/src/components/constants.ts]: I've updated

**Review by vittoriaThinkst — APPROVED** [review]

**Pull request merged**